src/test/rustdoc/decl_macro_priv.rs

```diff
@@ -2,7 +2,7 @@
 
 #![feature(decl_macro)]
 
-// @has decl_macro_priv/macro.crate_macro.html //pre 'pub(crate) macro crate_macro() {'
+// @has decl_macro_priv/macro.crate_macro.html //pre 'macro crate_macro() {'
```

Comment:
This looks wrong. It would be fixed by checking `pub(crate)` before checking `pub(self)`.

Reply:
Yep, fixed.

Reply:
I'm kind of surprised that the private macro isn't rendered as `pub(crate)` (which is what a private function in that position would do). Is that a quirk of declarative `macro`s?